src/ResourceManager/Batch/Commands.Batch.Test/Commands.Batch.Test.csproj

```diff
@@ -60,16 +60,15 @@
     <Reference Include="Microsoft.Azure.Gallery">
       <HintPath>..\..\..\packages\Microsoft.Azure.Gallery.2.6.2-preview\lib\net40\Microsoft.Azure.Gallery.dll</HintPath>
     </Reference>
-    <Reference Include="Microsoft.Azure.Management.Authorization">
+    <Reference Include="Microsoft.Azure.Management.Authorization, Version=0.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
This is a big problem - thgis will cause failures in upgrade scenarios - please look at the assembly and file version for your published nuget library